modules/Context.js

```diff
@@ -244,16 +243,16 @@ export class Context extends EventEmitter {
   }
 
 
-  loadTiles(projection, callback) {
+  loadTiles(callback) {
```

Comment:
No drawbacks at all!  This goes way back to the iD days - we went through a phase where we didn't like `context` and tried to limit its spread throughout the code.  This meant that service code (like `OsmService.js`) didn't actually have a reference to `context`.  We thought that it would be cleaner this way, but what ended up happening was that we just needed to pass more parameters around. 

Today we just accept that the application context is something useful that all the system components have access to - there are a lot of benefits to doing it this way, like being able to break up Rapid into modules, better control of how components start up, no more sneaky global components (like l10n and presets were a big ones of these).
, so we don't need to pass around references to things like `projection` (now `viewport`) that all the system components can get to..